<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Финансы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item><widget class="QPushButton" name="btnAddAccount"><property name="text"><string>Добавить счёт</string></property></widget></item>
    <item><widget class="QPushButton" name="btnDelAccount"><property name="text"><string>Удалить счёт</string></property></widget></item>
    <item><widget class="QPushButton" name="btnAddTx"><property name="text"><string>Добавить транзакцию</string></property></widget></item>
    <item><widget class="QPushButton" name="btnDelTx"><property name="text"><string>Удалить транзакцию</string></property></widget></item>
    <item><widget class="QPushButton" name="btnShowImage"><property name="text"><string>Показать изображение</string></property></widget></item>
    <item><widget class="QPushButton" name="btnShowChart"><property name="text"><string>Показать график</string></property></widget></item>

    <item><widget class="QLabel" name="lblAccountsTitle"><property name="text"><string>Счета</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
    <item><widget class="QTableWidget" name="tblAccounts"/></item>

    <item><widget class="QLabel" name="lblTransactionsTitle"><property name="text"><string>Транзакции</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
    <item><widget class="QTableWidget" name="tblTransactions"/></item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
